index.js

```diff
@@ -111,9 +111,35 @@ async function run(env, body, fs, core) {
     return result
   }
 
-  result = body
-    .trim()
-    .split("###")
+  function parseBody(body) {
+    let result = [];
+    let ignore = false;
+
+    body.split("\n").reduce((str, line, idx, arr) => {
+      // ``` Section must be ignored and not parsed as new section
+      if (line == "```")
```

Comment:
Please also account for instances when the context of the code block is specified. For example:

```
```md
```